lib/postdoc/client.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'chrome_remote'
+
+module Postdoc
+  class Client
+    attr_accessor :client
+
+    def initialize(port)
+      20.times { setup_connection_or_wait(port) && break }
+    end
+
+    def print_pdf_from_html(file_path,
+        header_template: false,
+        footer_template: false,
+        **options)
+      client.send_cmd 'Page.enable'
```

Comment:
I get: `undefined method `send_cmd' for nil:NilClass ` on this line in multiple projects when switching from the latest release to this branch. These use `render pdf: 'show'` in the controller, in the same way as `test/dummy` does.

Reply:
I'll look into it!